packages/trackerless-network/src/logic/StreamEntryPointDiscovery.ts

```diff
@@ -138,11 +134,11 @@ export class StreamEntryPointDiscovery {
 
     async storeSelfAsEntryPointIfNecessary(
         streamPartId: StreamPartID,
-        joiningEmptyStream: boolean,
+        avoidNetworkSplit: boolean,
```

Comment:
the method can analyze `stream.layer1!.getBucketSize() < this.config.networkSplitAvoidanceLimit` without us passing the booleann

Reply:
This change turns out to require a larger refactor related to adding an interface such as `ILayer1` which the layer1 DhtNodes implement. Keeping the code as is makes it more testable for now.